allauth/socialaccount/providers/battlenet/provider.py

```diff
@@ -5,17 +5,13 @@
 
 
 class BattleNetSocialAccountAdapter(DefaultSocialAccountAdapter):
```

Comment:
@pennersr Did you read my comment [here](https://github.com/pennersr/django-allauth/pull/1380#issuecomment-226943798)? It explains why I am still using the adapter.